lib/ReactViews/Custom/registerCustomComponentTypes.js

```diff
@@ -148,6 +150,15 @@ const registerCustomComponentTypes = function(terria) {
         }
         const url = defined(sources) ? sources[0] : undefined;
         const tableStructure = tableStructureFromStringData(getSourceData(node, children));
+
+        const colors = splitStringIfDefined(node.attribs['colors']);
+        if (defined(colors)) {
+            const getColorCallback = function(columnId) {
+                return colors[columnId];
+            };
+            tableStructure.getColorCallback = getColorCallback;
```

Comment:
This new attribute `colors` only works if there is a `tableStructure`, ie. if actual data is provided in the chart tag. We should at least note that. But it would be even better if there's a way to make it work more generally.